src/app/options/qgsuserprofileoptions.cpp

```diff
@@ -0,0 +1,162 @@
+/***************************************************************************
+    qgsuserprofileoptions.cpp
+    -----------------
+    begin                : February 2023
+    copyright            : (C) 2023 by Yoann Quenach de Quivillic
+    email                : yoann dot quenach at gmail dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "qgssettings.h"
+#include "qgsapplication.h"
+#include "qgisapp.h"
+#include "qgsuserprofilemanager.h"
+
+#include "qgsuserprofileoptions.h"
+
+//
+// QgsUserProfileOptionsWidget
+//
+
+QgsUserProfileOptionsWidget::QgsUserProfileOptionsWidget( QWidget *parent )
+  : QgsOptionsPageWidget( parent )
+{
+  setupUi( this );
+
+  auto manager = QgisApp::instance()->userProfileManager();
```

Comment:
can you avoid the use of "auto" in this class? QGIS coding standard is to only use auto for complex types (like iterators).

Reply:
Ok